src/defold/Resource.hx

```diff
@@ -74,6 +106,26 @@ extern class Resource {
          * `status` Whether or not the resource was successfully stored.
     **/
     static function store_resource<T>(manifest_reference:ResourceManifestReference, data:String, hexdigest:String, callback:T->String->Bool->Void):Void;
+
+    /**
+        Constructor-like function with two purposes:
+
+        - Load the specified resource as part of loading the script
+        - Return a hash to the run-time version of the resource
+
+        **Note:** This function can only be called within `@property()`.
+    **/
+    static function texture(path:Hash):ResourceReference;
+
+    /**
+        Constructor-like function with two purposes:
+
+        - Load the specified resource as part of loading the script
+        - Return a hash to the run-time version of the resource
+
+        **Note:** This function can only be called within `@property()`.
```

Comment:
I have to look at how this is supposed to be used, because if it's something for the `go.property` calls, we have to add this to the script macros.

Reply:
The generated code should look like this:

```lua
go.property(“z”, resource.atlas("/data/sprites.atlas"))
```